Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -5,6 +5,8 @@
 # common utilities
 ##############################################################################
 def _swapOfflineBSwithOnline(process):
+    process.BeamSpotESProducer = cms.ESProducer("OnlineBeamSpotESProducer")
```

Comment:
```suggestion
    import RecoVertex.BeamSpotProducer.onlineBeamSpotESProducer_cfi as _mod
    BeamSpotESProducer = _mod.onlineBeamSpotESProducer.clone()
```
please use cfipython default explicitly.

ah, but that's a future comment relative to #35333

Reply:
so this PR unfortunately needs more work, in particular we need to decide what to do for the MC GlobalTags (that now that the modifier is working correctly need to be updated)
I'd propose to merge  #35333 first.

Reply:
fixed in the last push.